source/getting-started/index.md

```diff
@@ -37,6 +38,16 @@ installing Homebrew, run `brew install node` to install Node.js.
 Once you've got Node.js installed, re-run `node --version` to verify your
 computer's setup.
 
+If you already have npm and Node.js installed, it's a good idea to make sure you're on the latest verson of npm since it gets updated more frequently than Node.js does. 
+
+To update npm, run:
+
+```bash
+sudo npm install npm -g
```

Comment:
Telling people to use sudo generally strikes me as a bad idea without the proper context and explanation, which I don't think we should get into here.